app_dart/test/request_handlers/get_green_commits_test.dart

```diff
@@ -86,8 +93,8 @@ void main() {
 
       expect(result.length, 2);
       expect(result, <String>[
-        'd5b0b3c8d1c5fd89302089077ccabbcfaae045e4',
-        'ea28a9c34dc701de891eaf74503ca4717019f829',
+        commit2.sha!,
```

Comment:
Minor nit: speaking of null safety, should we make commit1/2 null safe and not have the